### ISSUE TITLE
Weird ship detecting

### ISSUE THREAD

**Issue by craft90** [LemADEC/WarpDrive#231]
So I'm new to this mod. I followed the Getting Started wiki post. However I'm puzzled as for how these directions works. There are no clear indications on neither the ship coren or the controller on how it reads directions. Apparently left of the ship core isn't actually left but in the WEST direction. That could have been much clearer.  Because a broken ship kinda looks dumb and I imagine kills you. Like look at this; I specified something like back as 8 blocks in the dimensions config and this happened. And I specified left as 5 blocks but it only detected 2 blocks
![2017-01-06_03 09 55](https://cloud.githubusercontent.com/assets/12762624/21705158/76980d8c-d3be-11e6-9d87-2db463c4914c.png)

I also noticed that the entry is outdated. S doesn't lead to warp. Now it's J.

**Comment by LemADEC** [comment]
Cardinal direction are irrelevant because you can turn your ship around.
The ship core is the center, the controller is the front.
Or if you prefer, the controller is by definition placed in front.

**Comment by craft90** [comment]
Yeah I figured it out eventually. But from the tutorial is seemed like the front was which the computer screen faced. Which I misunderstood as meaning that the screen had to face your intended front (which screwed my initial ship up). It could have been clearer. In my case it turned out front was south (which feels pretty odd. But whatever) once I finally managed to pin the issue down. So yeah it was the direction the controller pointed at. I guess there's no easy way of explaining it. Glad I tested it in creative instead of going straight on it in survival. It feels a little odd to have the controls on the side rather than facing you.
